NovelRT.Sdk/TemplateFiles/CMakeTemplate/src/PROJECT_NAME/CMakeLists.txt

```diff
@@ -1,32 +1,31 @@
-﻿include(CopyBuildProducts)
-include(CopyDirectoryRaw)
+﻿include(CopyDirectoryRaw)
 
 find_package(novelrt ${###PROJECT_NAME###_NOVELRT_VERSION} REQUIRED)
 
 set(SOURCES
     main.cpp
   )
 
-add_executable(###PROJECT_NAME### ${SOURCES})
+add_executable(###PROJECT_NAME###_Game ${SOURCES})

```

Comment:
Target names cannot match the project name - targets _must_ be unique in this case.